tools/bin/hawqextract

```diff
@@ -558,13 +558,15 @@ def create_opt_parser(version):
     parser.add_option('-o', '--output', help="the output metadata file, defaults to stdout", metavar='FILE')
     parser.add_option('-W', action='store_true', dest='use_getpass', help="force password authentication")
     parser.add_option('-v', '--verbose', action='store_true')
+    parser.add_option('-l', '--logdir', dest='logDir', help="Sets the directory for log files")
     return parser
 
 
 def main(args=None):
     parser = create_opt_parser('%prog version $Revision: #1 $')
 
     options, args = parser.parse_args(args)
+    setup_tool_logging(EXECNAME, getLocalHostname(), getUserName(), logdir=options.logDir)
```

Comment:
If user doesn't give a logdir, the default value of `options.logDir` is?
Anyway, just make sure there is no problem.

Thanks!

Reply:
@interma - setup_tool_logging  defaults the value of logdir to None and handles it inside the function as seen from [here](https://github.com/apache/incubator-hawq/blob/e082d3adbb89a9e5ccb341da324ea9514cff5a35/tools/bin/gppylib/gplog.py#L130). I tested it locally and it works fine.

Reply:
Thanks! 

LGTM.